lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1401,118 +1440,37 @@ export class UserAgentApplication {
         }
     }
 
+    /**
+     * 
+     * @hidden
+     * 
+     * Searches the token cache for an access token that matches the request parameters and returns it as an AuthResponse.
+     * 
+     * @param serverAuthenticationRequest 
+     * @param account 
+     * @param scopes 
+     */
     private getCachedAccessToken(serverAuthenticationRequest: ServerRequestParameters, account: Account, scopes: string[]): AuthResponse {
-        // filter by clientId and account
-        const tokenCacheItems = this.cacheStorage.getAllAccessTokens(this.clientId, account ? account.homeAccountIdentifier : null);
+        this.logger.verbose("Getting all cached tokens of type Access Token");
+        const tokenCacheItems = this.cacheStorage.getAllTokensByType(this.clientId, account ? account.homeAccountIdentifier : null, ServerHashParamKeys.ACCESS_TOKEN);
```

Comment:
Reason for this change? `getAllAccessTokens` seemed more descriptive

Reply:
Changed it hoping I'd be able to factor the beginning of these methods into a single shared method for IdTokens and AccessTokens, but it turned out to not be trivial. Switched back to using the type-specific methods.